.claude/scripts/release_testing/precheck_ecrc.py

```
@@ -0,0 +1,264 @@
+    parser.add_argument("--verbose", action="store_true",
+                        help="Show extra detail lines")
+    args = parser.parse_args()
+
+    release_type = args.type.upper()
+
+    # Step 1: Discover latest EC/RC
+    if args.version:
+        logger.info("Fetching specific release: %s", args.version)
+        try:
+            release = release_controller.get_specific_release("4-dev-preview", args.version)
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)